### ISSUE TITLE
[ic-stepper] color-contrast a11y violation for light stepper

### ISSUE THREAD

**Issue by MI6-286** [mi6/ic-ui-kit#2050]
## Summary of the bug 
A11y testing of a light ic-stepper rendered with background #2c2f34 has failed due to a color-contrast violation (see below). 

## 🪜 How to reproduce 
Uncomment the commented out a11y test in the ic-stepper Cypress tests and run test.
See the error.
Alternatively, view the results on the accessibility tab on the IcStepper/Light example on Storybook.

## 📸 Screenshots or code 
Elements with the violation highlighted in screenshot below:
![image](https://github.com/user-attachments/assets/ac6d8770-470e-46a5-bbc2-99c219c13372)
![image](https://github.com/user-attachments/assets/2f9d2936-b252-44dd-b43f-a79a21e8c086)

## 🖥 📱 Device 
- Type:  Cypress a11y testing

## 🧐 Expected behaviour 
There should be no accessibility violations. To check this please uncomment the a11y test in the ic-stepper Cypress tests.

Make font colour of steps dark-font colour

**Comment by GCHQ-Developer-299** [comment]
Holding off on request of design team, this will be covered by Dark Mode work

**Comment by GCHQDeveloper618** [comment]
we are currently using #121212 as our dark mode background colour against the components as it was the best colour to use from research we conducted whereas the colour used above is #2c2f34

**Comment by MI6-255** [comment]
Wait for the ticket to update all the colour token values, which will fix this as the blue is fixed in Figma. The grey disabled state is a false negative
